terraform/projects/app-ecs-services/templates/nginx-auth-proxy.conf.tpl

```diff
@@ -1,43 +1,72 @@
 server {
-  listen 9090;
+  listen 8181 default_server;
+
+  location /health {
+    return 200 "Static health check";
+  }
```

Comment:
you said in the PR description that we want to return a 404, but there's nothing here that explicitly sets up that behaviour.  If this is something we care about, do we want to have an explicit 404 here?

Reply:
The default behaviour in this case will be to return 404